install-and-configure/install/multi-cluster/federated-etl/aggregator.md

```diff
@@ -0,0 +1,143 @@
+# Kubecost Aggregator
+
+{% hint style="info" %}
+Kubecost Aggregator is a beta feature.
+{% endhint %}
+
+Aggregator is a new, experimental backend for Kubecost. It is used in a [Federated ETL](/install-and-configure/install/multi-cluster/federated-etl/federated-etl.md) configuration without Thanos, replacing the [Federator](/install-and-configure/install/multi-cluster/federated-etl/federated-etl.md#other-components) component.
+Aggregator serves a critical subset of Kubecost APIs, but will eventually be the default model for Kubecost and serve all APIs. Currently, Aggregator all major monitoring and savings APIs, and also budgets and reporting.
+
+{% hint style="info" %}
+Existing documentation for Kubecost APIs will use endpoints for non-Aggregator environments unless otherwise specified, but will still be compatible after configuring Aggregator.
+{% endhint %}
+
+Aggregator is designed to accommodate queries of large-scale datasets, including reduced load times and potential errors in the UI. It is not designed to introduce new functionality; it is meant to improve functionality at scale.

```

Comment:
What is "and potential errors in the UI" supposed to mean in this sentence?

Reply:
From what I've heard, errors where high volume of data is unable to be rendered/displayed. If this is not accurate to common user troubleshooting, I'll reword it (or can reword it period to be more specific)

Reply:
Ah, I see. Perhaps:

> Aggregator is designed to accommodate queries of large-scale datasets by improving API load times and reducing UI errors.